packages/non-core/bundle-visualizer/server.js

```diff
@@ -1,4 +1,4 @@
-import assert from "assert";
+import assert from "node:assert";
 import { readFileSync as fsReadFileSync } from "fs";
```

Comment:
`fs` should probably also be `node:fs`

Reply:
I wonder if this is even necessary, because `meteor-node-stubs` should be responsible to resolve and transform these. From what I found, it's not 100% required but good practice to use the scoped import for node core modules.